debug_probes.md

```diff
@@ -2,19 +2,28 @@
 
 ## Raspberry Pi Pico
 
-  You can use a second Pico as your debugger.
-
-  - Download this file: https://github.com/majbthrd/DapperMime/releases/download/20210225/raspberry_pi_pico-DapperMime.uf2
-  - Boot the Pico in bootloader mode by holding the bootset button while plugging it in
-  - Open the drive RPI-RP2 when prompted
-  - Copy raspberry_pi_pico-DapperMime.uf2 from Downloads into RPI-RP2
-  - Connect the debug pins of your CMSIS-DAP Pico to the target one
-      - Connect GP2 on the Probe to SWCLK on the Target
-      - Connect GP3 on the Probe to SWDIO on the Target
-      - Connect a ground line from the CMSIS-DAP Probe to the Target too
-
-  If you have good wiring between your Pico's, you can instead use rust-dap for faster programming:
-  https://raw.githubusercontent.com/9names/binary-bits/main/rust-dap-pico-ramexec-setclock.uf2
+You can use a second Pico as your debugger.
+
+Download one of these firmware files:
+
+- [picoprobe.uf2](https://github.com/raspberrypi/picoprobe/releases/download/picoprobe-cmsis-v1.0.1/picoprobe.uf2) -
+  Official raspberrypi probe firmware supporting CMSIS-DAP.
+- [raspberry_pi_pico-DapperMime.uf2](https://github.com/majbthrd/DapperMime/releases/download/20210225/raspberry_pi_pico-DapperMime.uf2) -
+  Based upon an older version of the CMSIS-DAP sources.
+- [rust-dap-pico-ramexec-setclock.uf2](https://raw.githubusercontent.com/9names/binary-bits/main/rust-dap-pico-ramexec-setclock.uf2) -
+  If you have good wiring between your Pico's, this firmware will give faster
+  programming.
+
+Then:
+
+1. Boot the Pico in bootloader mode by holding the bootset button while plugging
```

Comment:
We should try to match the language used in the [Pico "getting started"](https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf#%5B%7B%22num%22%3A13%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C115%2C841.89%2Cnull%5D) guide where possible:

```suggestion
1. Put the Pico into USB Mass Storage Mode by holding the BOOTSEL button while connecting it to your computer with a USB cable
```

Reply:
I just copied the existing text. But the suggestion is indeed clearer, except that the following line needs fixing as well.